File: src/main/java/com/googongill/aditory/external/chatgpt/ChatGptConfig.java
package com.googongill.aditory.external.chatgpt;

import lombok.Getter;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.context.annotation.Configuration;

@Getter
@Configuration
public class ChatGptConfig {
    @Value("${chat-gpt.url}")
    private String url;
    @Value("${chat-gpt.key}")
    private String apiKey;
    @Value("${chat-gpt.model}")
    private String model;
    @Value("${chat-gpt.max_token}")
    private Integer maxToken;
    @Value("${chat-gpt.temperature}")
    private Double temperature;
    @Value("${chat-gpt.topP}")
    private Double topP;
}

File: src/main/java/com/googongill/aditory/external/chatgpt/ChatGptConfig.kt
package com.googongill.aditory.external.chatgpt

import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.Configuration

@Configuration
class ChatGptConfig (
    @Value("\${chat-gpt.url}")
    val url: String,
    @Value("\${chat-gpt.key}")
    val apiKey: String,
    @Value("\${chat-gpt.model}")
    val model: String,
    @Value("\${chat-gpt.max_token}")
    val maxToken: Int,
    @Value("\${chat-gpt.temperature}")
    val temperature: Double,
    @Value("\${chat-gpt.topP}")
    val topP: Double
)
